.github/workflows/lint_black.yaml

```diff
@@ -16,10 +16,10 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v1
         with:
-          python-version: 3.8
+          python-version: 3.10.13
```

Comment:
Any particular reason to choose this version?

Really small nitpick here, but I would keep the `3.8` so we can ensure that developing `rocketpy` is also possible with the oldest supported `Python` version.  If there is a meaningful reason to not use the `3.8`, I am fine with the change.

Reply:
You are right, I fixed it on my last commit. [374eefd](https://github.com/RocketPy-Team/RocketPy/pull/500/commits/374eefd3fe1627d53a10db98fcdc9a6a66b9b4d7)

I didn't pay attention, but black may give different results depending on the python version.